tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -918,6 +920,41 @@ public function it_should_create_mime_types_for_allowed_sizes_only_via_global_va
 		$this->assertImageNotHasSizeSource( $attachment_id, 'not_allowed_size_200x150', 'image/webp' );
 	}
 
+	/**
+	 * Test image quality for image conversion.
+	 *
+	 * @test
+	 */
+	public function it_should_set_quality_with_image_conversion() {
```

Comment:
This test is useful for an integration test, however it's almost impossible to validate here that the logic in the underlying function `webp_uploads_modify_webp_quality()` is correct. I would suggest we add another (simpler) test that calls `webp_uploads_modify_webp_quality()` directly, passing a specific default and MIME type to it so that we can really cover the internals of that function and that they work as expected.

Reply:
I have pushed new simple test to test `webp_uploads_modify_webp_quality()` function. It is actually really simple just calling the function with quality and check whether it return correct quality for mimes or not.